std_msgs/Float32MultiArray y
std_msgs/Float32MultiArray z
std_msgs/Float32MultiArray r
std_msgs/Float32MultiArray b